shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -70,7 +70,7 @@ public void registerCompletion(final Token root,
   @Override
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
       throws AccumuloException, AccumuloSecurityException, TableNotFoundException, IOException,
-      ClassNotFoundException, NamespaceNotFoundException {
```

Comment:
Meant to put this in its own commit. We arent throwing this anywhere so i removed it to clean up a a bit.